research-software-directory/RSD-as-a-service
# Connect frontend to backend

# Connect frontend to postgREST backend

Handles #52 

Changes proposed in this pull request:

* Create local development and local production env files
* Connect software and project page to postgrest backend
* Upgrade tailwind css to new version (v3.0)

How to test:
* Copy `.env.local.example` to `.env.production.local`
* Start the solution `docker-compose up`
* Navigate to software page, you should see a list of software names similar to image below
* You can use navigation on the right, change number of items per page etc.
* On the projects page similar functionality is implemented

![image](https://user-images.githubusercontent.com/9204081/145954007-426c23ae-f95f-4521-a1e3-be1a6b0bd957.png)
